app/assets/stylesheets/content/_forms.sass

```diff
@@ -535,6 +535,11 @@ fieldset.form--fieldset
   & > [data-tooltip], & > [class^="tooltip--"]
     padding-top: 0.6rem
 
+.form--field
+  & .-break-words > .form--label
+    text-overflow: ellipsis
+    overflow: hidden
```

Comment:
Still think that it would be better to generally apply the rules and rather define exceptions to this for the cases you mentioned. But granted, this might become a little too big for such a fix.

In any case, the rules are not applied because of the whitespace `& .-break-words`. This should rather be `&.-break-words`.

Additionally, I don't think that the placement of the code is right as we already have a form--field [top-level statement](https://github.com/opf/openproject/blob/dev/app/assets/stylesheets/content/_forms.sass#L207). On the other hand that might not matter as I think that the modifier class should rather be applied directly to the label as it only affects the label while doing nothing to change the dimensions of it. However, that will probably require changes to more places in the code. :(